src/EvoSC.Modules/ModuleManager.cs

```diff
@@ -304,7 +419,7 @@ private async Task RegisterModuleConfigAsync(IEnumerable<Assembly> assemblies, C
                     if (!type.IsInterface)
                     {
                         _logger.LogError("Settings type {Type} must be an interface", type);
```

Comment:
```suggestion
                        _logger.LogError($"Settings type {type} must be an interface");
```
Would it make more sense to use the same string interpolation as in the throw below here?
Consistency across the board in this regard goes a long way imho.

Reply:
The way I did it is the official pattern for the logging interface. String interpolation causes problems and should not be used.